### PR TITLE
[StyleCop] Fix all warnings in Japanese\TimeParser file

### PR DESCRIPTION
**Pull request by ceciliaavila**
- Classes were separated into different files.
- Reordered using statements and methods.
- Added spaces and trailing commas when needed.

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeFunctions.cs [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Japanese;
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime.Japanese
+{
+
```
Comment (Aliandi) [inline .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeFunctions.cs]: Remove space

File: .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeResult.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Japanese;
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime.Japanese
+{
+
```
Comment (Aliandi) [inline .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeResult.cs]: Remove space

**Review by Aliandi — CHANGES_REQUESTED** [review]
There are extra spaces in the new files after the namespace.

File: .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeFunctions.cs (hunk 1 above)
Comment (ceciliaavila) [reply .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeFunctions.cs]: Done!

File: .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeResult.cs (hunk 2 above)
Comment (ceciliaavila) [reply .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimeResult.cs]: Done!

**Review by Aliandi — APPROVED** [review]

**Review by JuanAr — APPROVED** [review]
LGTM

**Pull request closed without merging**